Index: Makefile
===================================================================
RCS file: /cvs/ports/devel/p5-Test-File/Makefile,v
retrieving revision 1.6
diff -u -p -r1.6 Makefile
--- Makefile	22 Mar 2013 16:59:04 -0000	1.6
+++ Makefile	20 Jan 2016 04:45:45 -0000
@@ -2,7 +2,7 @@
 
 COMMENT=		test file attributes
 
-DISTNAME=		Test-File-1.34
+DISTNAME=		Test-File-1.44
 CATEGORIES=		devel
 
 # perl
Index: distinfo
===================================================================
RCS file: /cvs/ports/devel/p5-Test-File/distinfo,v
retrieving revision 1.2
diff -u -p -r1.2 distinfo
--- distinfo	22 Mar 2013 16:59:04 -0000	1.2
+++ distinfo	20 Jan 2016 04:45:45 -0000
@@ -1,2 +1,2 @@
-SHA256 (Test-File-1.34.tar.gz) = XGn8L765kIZtiUBzSj9eHjejEv3iDUXhXzlvgbPs7qs=
-SIZE (Test-File-1.34.tar.gz) = 18771
+SHA256 (Test-File-1.44.tar.gz) = QqSDVN4+nr8cThVffYvQRcIlG8VsrTua/Cye50BLbMk=
+SIZE (Test-File-1.44.tar.gz) = 26498
